pdns/dnsdistdist/Makefile.am

```diff
@@ -23,21 +23,24 @@ EXTRA_DIST=dnslabeltext.rl \
 	   .version \
 	   contrib
 
-bin_PROGRAMS = dnsdist
+bin_PROGRAMS = dnsdist test-dnsdist_cc
```

Comment:
In this case I think the tests will actually move into `unit-tests`. Also I don't mind building test binaries always if they don't have extra dependencies.